Project/js/menu.js

```diff
@@ -1,5 +1,8 @@
+const menu = document.querySelector(".main-nav__list--responsive").classList;
```

Comment:
It would be better to just store `document.querySelector(".main-nav__list--responsive")` as this is expensive operation, then you will have access to all of it properties